assets/tests/components/detours/detoursListPage.openDetour.test.tsx

```diff
@@ -88,4 +62,36 @@ describe("Detours Page: Open a Detour", () => {
     // Finally, check snapshot
     await waitFor(() => expect(baseElement).toMatchSnapshot())
   })
+
+  test("renders detour details in an open drawer on mobile", async () => {
+    mockScreenSize("mobile")
+
+    jest.mocked(fetchDetours).mockResolvedValue(Ok(detourListFactory.build()))
+
+    jest
+      .mocked(fetchDetour)
+      .mockResolvedValue(Ok(detourStateMachineFactory.build()))
+
+    const { baseElement } = render(<DetourListPage />)
+
+    await userEvent.click(await screen.findByText("Headsign Z"))
+
+    await waitFor(() => expect(baseElement).toMatchSnapshot())
+  })
+
+  test("detour details drawer is collapsible on mobile", async () => {
+    mockScreenSize("mobile")
+
+    jest.mocked(fetchDetours).mockResolvedValue(Ok(detourListFactory.build()))
+
+    jest
+      .mocked(fetchDetour)
+      .mockResolvedValue(Ok(detourStateMachineFactory.build()))
+
+    const { baseElement } = render(<DetourListPage />)
+
+    await userEvent.click(await screen.findByText("Headsign Z"))
+    await userEvent.click(await screen.findByTitle("Collapse"))
+    await waitFor(() => expect(baseElement).toMatchSnapshot())
```

Comment:
instead of using snapshots, can we instead expect that the button title changes, or something else?

Reply:
(backlink) https://github.com/mbta/skate/pull/2844/files#r1795923279

Reply:
For this one, totally agree that mostly I should focus on the button change; it's overkill to do a whole snapshot match when we just checked the snapshot in the last test.

But for [the first test](https://github.com/mbta/skate/pull/2844#discussion_r1795923279), I'm thinking it might be nice to keep the snapshot because I want to check that the backdrop was applied, the drawer is open, and the correct classnames are there for mobile view. But I could check those other ways if you think that's still preferable to using a snapshot?

Reply:
Snapshots are somewhat inscrutable for review, so I would prefer it to use other methods. But 1 snapshot is better than 2.

Reply:
Unfortunately, the things I most want to check don't have easily queried values. The backdrop is a button, but has no name or alternative selector other than its class. The openness of the drawer is difficult to check period (given jest only attends to the DOM and not the framing of an element in the viewport. So... I guess I'll stick with 1 snapshot for now?